lms/static/scripts/frontend_apps/utils/api.ts

```diff
@@ -271,6 +289,10 @@ export function usePolledAPIFetch<T = unknown>({
     timeout.current = null;
   }, [_clearTimeout]);
 
+  const refreshing = useRef(false);
```

Comment:
The meaning of this may be unclear. Perhaps `refreshScheduled` would make it clearer that this flag indicates whether `retry` has been called, rather than whether the request is in-flight, since the flag is immediately cleared on the next render.